db/migrate/20150416231106_add_source_to_webhook.rb

```diff
@@ -0,0 +1,12 @@
+class AddSourceToWebhook < ActiveRecord::Migration
+  def up
+    add_column :webhooks, :source, :string, null: false, default: 'any_ci'
+    change_column_default :webhooks, :source, nil
```

Comment:
I guess I was suggesting make it not-null completely. Any reason to have a null source?